templates/prescription/general_edit.html

```diff
@@ -256,10 +265,24 @@
 {$ENDING_JAVASCRIPT}
 </script>
 </form>
+<!-- Has to be loaded after the form -->
```

Comment:
Why did you move these to the bottom? Rec moving them to the top where include the css stuff. This is likely why getting the following js errors when open the prescription Add gui:
![screenshot from 2017-09-02 21-18-38](https://user-images.githubusercontent.com/278968/30000518-92cc8d7a-9024-11e7-9489-95a41f1f0037.png)

Reply:
I have not move anything in this area.

Reply:
AAAARRRRRRR!!! I keep fixing this and it keeps getting broken :-(

Reply:
jquery-ui.min.css
This file is missing.

Reply:
Moved to the top and it works still so leaving and removing the note.